From a318fcd8d5110a3271ce7c4a4408ab363372451e Mon Sep 17 00:00:00 2001
From: Fabio Berton <fabio.berton@ossystems.com.br>
Date: Thu, 28 Apr 2016 17:32:56 -0300
Subject: [PATCH] Remove include sys/unistd.h
Organization: O.S. Systems Software LTDA.

Remove include <sys/unistd.h> in alternatives.c to avoid breaks build
with musl.

Upstream-Status: Pending

Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
---
 alternatives.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/alternatives.c b/alternatives.c
index 3a012d0..62fe539 100644
--- a/alternatives.c
+++ b/alternatives.c
@@ -22,7 +22,6 @@
 #include <stdio.h>
 #include <string.h>
 #include <sys/stat.h>
-#include <sys/unistd.h>
 #include <unistd.h>
 
 #define	FLAGS_TEST	(1 << 0)
-- 
2.1.4

